src/picknik_ur_site_config/objectives/take_snapshot.xml

```diff
@@ -1,14 +1,14 @@
-<?xml version="1.0"?>
+<?xml version='1.0' encoding='UTF-8'?>
 <root BTCPP_format="4" main_tree_to_execute="Take Snapshot">
-  <!-- ////////// -->
+  <!--//////////-->
   <BehaviorTree ID="Take Snapshot" _description="Take a point cloud snapshot and add to collision scene" _favorite="true">
     <Control ID="Sequence">
       <Action ID="SwitchUIPrimaryView" primary_view_name="Visualization"/>
-      <Action ID="GetPointCloud" topic_name="/wrist_mounted_camera/depth/color/points" message_out="{point_cloud}"/>
+      <Action ID="GetPointCloud" topic_name="/camera/wrist_mounted_camera/depth/color/points" message_out="{point_cloud}" timeout_sec="15.000000"/>
```

Comment:
Is this topic name change correct?

Reply:
Yes, all of these in all the ur packages should be updated actually, just didn't want to touch too much here.

Reply:
Yes, all of these in all the ur packages should be updated actually, just didn't want to touch too much here.

Reply:
Yes, this was required during our testing on hardware. Not sure of exact variations between 5.0 and main tbh. I've updated all the topics to have the `/camera` prefix now.